lib/livebook/notebook/cell/input.ex

```diff
@@ -96,6 +96,7 @@ defmodule Livebook.Notebook.Cell.Input do
   @spec default_props(type()) :: props()
   def default_props(type)
 
+  def default_props(:checkbox), do: %{label: ""}
```

Comment:
I don't think the checkbox needs label, because we already have input name that should be descriptive by itself.